operator/v1/types_network.go

```diff
@@ -507,6 +507,26 @@ type GatewayConfig struct {
 	// The supported values are "Restricted" and "Global".
 	// +optional
 	IPForwarding IPForwardingMode `json:"ipForwarding,omitempty"`
+	// V4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by
```

Comment:
Do you expect in the future to add any other config to this, that might be split into IPv4 and IPv6 versions? Wondering if creating some structure now will make this easier to consume in the future?
```
gatewayConfig:
  ipForwarding: Restricted
  ipv4:
    internalMasqueradeSubnet: ...
  ipv6:
    internalMasqueradeSubnet: ...
```

Reply:
I wonder if we should go ahead and do it anyway, just in case, IMO it's cleaner and more future proof, WDYT?

Reply:
Agree on the future proof aspect. I dont currently see a scenario in which we would be adding anything to the gateway config, but that could change, so I can make this change